platforms/ios/demo/src/MapViewController.m

```diff
@@ -18,6 +18,15 @@ - (void)addAlert:(NSString *)message withTitle:(NSString *)title;
 
 @implementation MapViewControllerDelegate
 
+- (void)mapView:(TGMapViewController *)mapView didCompleteSceneUpdatesWithErrors:(NSArray<TGSceneUpdateStatus *> *)updateStatuses
+{
+    for (TGSceneUpdateStatus* updateStatus in updateStatuses) {
+        NSLog(@"Scene update error for update with path %@ and value %@",
```

Comment:
>As I mentioned for android interface, instead of logging

Agreed on NSError, but this is the demo app where it is fine to log to show example usage.

>It would be neat to know what failed and what didn't, yep. 

Please look at the interface in the public framework and usage of `TGSceneUpdateStatus`, here it is the demo app. Also, please mention how `NSError` would be best used for your needs.